src/monarch_ingest/ingests/dictybase/gene.py

```
@@ -1,17 +1,14 @@
-from koza.cli_utils import get_koza_app
-
+import koza
 from biolink_model.datamodel.pydanticmodel_v2 import Gene
 
-koza_app = get_koza_app("dictybase_gene")
-taxon_labels = koza_app.get_map("taxon-labels")
-
-in_taxon = "NCBITaxon:44689"
-in_taxon_label = taxon_labels[in_taxon]['label'] if in_taxon in taxon_labels else "Dictyostelium discoideum"
 
-while (row := koza_app.get_row()) is not None:
+@koza.transform_record()
+def transform_record(koza_transform, row):
+    in_taxon = "NCBITaxon:44689"
+    in_taxon_label = "Dictyostelium discoideum"
```

Previous version attempted to pull the taxon label dynamically from a hardcoded taxon id (NCBITaxon:44689).
This version just defaults to hardcoded version of both taxon_id and taxon_label.
Fine as is, but maybe nice to retain old functionality at some point in case the taxon label changes (can't imagine this would ever be the case though...)